pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py

```diff
@@ -0,0 +1,546 @@
+import os
+import time
+import shutil
+import argparse
+import collections.abc
+import gdown
+
+import numpy as np
+from sklearn.metrics import cohen_kappa_score
+
+import torch
+import torch.nn as nn
+from torch.cuda.amp import GradScaler, autocast
+
+from torch.utils.tensorboard import SummaryWriter
+from torch.utils.data.distributed import DistributedSampler
+from torch.utils.data.dataloader import default_collate
+
+import torch.distributed as dist
+import torch.multiprocessing as mp
+
+from monai.data import Dataset, load_decathlon_datalist
+from monai.data.image_reader import WSIReader
+from monai.metrics import Cumulative, CumulativeAverage
+from monai.transforms import Transform, Compose, LoadImageD, RandFlipd, RandRotate90d, ScaleIntensityRangeD, ToTensord
+from monai.apps.pathology.transforms import TileOnGridd
+from monai.networks.nets import milmodel
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser(description="Multiple Instance Learning (MIL) example of classification from WSI.")
+    parser.add_argument(
+        "--data_root", default="/PandaChallenge2020/train_images/", help="path to root folder of images"
```

Comment:
Here and elsewhere, can you change the underscore (`_`) in option names to dash (`-`)? It is easier (and more common way) to pass commandline args with dashes and argparse will automatically convert them to underscore for the variable name.
```suggestion
        "--data-root", default="/PandaChallenge2020/train_images/", help="path to root folder of images"
```

Reply:
argparse doesn't work well with dashes in names, it will be very hard to use it
https://stackoverflow.com/questions/12834785/having-options-in-argparse-with-a-dash

Reply:
@myron What you are referring to is positional arguments but here you have optional arguments starting with `--`. For `argparse` "Any internal - characters will be converted to _ characters to make sure the string is a valid attribute name."
https://docs.python.org/dev/library/argparse.html#dest

Reply:
I prefer to keep underscores in the name, it's explicit, and there will be no conversion between the definition and internal use.  I think it's perfectly fine to keep underscores

Reply:
Alright!